include/swift/SIL/SILValue.h

```diff
@@ -1037,6 +1037,8 @@ class Operand {
   /// guaranteed scope.
   bool isLifetimeEnding() const;
 
+  bool isConsuming() const;
```

Comment:
Quick drive by (need to read more). But this needs a comment to express the difference with isLifetimeEnding().